scripts/zones/RuAun_Gardens/mobs/Seiryu.lua

```diff
@@ -16,6 +16,8 @@ entity.onMobInitialize = function(mob)
     mob:addMod(xi.mod.DOUBLE_ATTACK, 10)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
     mob:setMobMod(xi.mobMod.MAGIC_COOL, 35)
+    -- TP move about every 9 seconds without TP feed
+    mob:setMod(xi.mod.REGAIN, 750)
```

Comment:
Isnt this gonna make it spam TP moves under a certain HP threshold? We sure this is right?

Reply:
> Isnt this gonna make it spam TP moves under a certain HP threshold? We sure this is right?

Yeah good point, I adjusted the code now so when it is under 25% its regain is set lower.